### ISSUE TITLE
memory leak

### ISSUE THREAD

**Issue by zhuxiujia** [vlang/v#3897]
```
// blog.v
module main

import (
	vweb
)

pub struct App {
mut:
	vweb vweb.Context
}

fn main() {
	vweb.run<App>(8080)
}

fn (app mut App) index() {
	app.vweb.text('Hello, world from vweb!')
}

pub fn (app &App) init() {}
pub fn (app &App) reset() {}
```

* v run http.v
building-a-simple-web-blog-with-vweb.md

* in windows10 request  http://127.0.0.1:8080/   every request this,  the  The memory grows by 0.1 MB

* this is rust request test, also you can use go,java or other lang to request this
``` rust
use hyper::Uri;
use rsd::http_client::{fetch_url, fetch_url_client};
use rsd::time_util::count_time_tps;
use std::time::{SystemTime, Duration};
use std::thread::sleep;


//cargo.exe run --release --color=always --package example --bin benchmark
#[tokio::main]
pub async fn main(){
    let client=hyper::Client::new();
    let s=fetch_url_client("POST","http://127.0.0.1:8080/","",&client).await.unwrap();
    println!("{}",s);


    let total=100000;
    let now=SystemTime::now();
    for _ in 0..total{
         sleep(Duration::from_secs(1));
         for _ in 0..100{
             fetch_url_client("POST","http://127.0.0.1:8080/","",&client).await.unwrap();
         }
    }
    count_time_tps(total,now);
}

```

**Comment by antonkatz** [comment]
I can confirm that this is still happening, albeit at 4K per request, not 100K.

V 0.1.29 3b5d562 on Mac

**Comment by smartiniOnGitHub** [comment]
Weeks ago (with previous V version) I had the same behavior (memory leak) doing a test of concurrent load using the utility Apache Bench ('ab' executable, part of the Apache Web Server), and even with the load testing utility WRK (it seems more accurate); all running in Ubuntu 20.04 LTS. 
I think it's still present in current V version.
Hope this helps simplifying some tests, otherwise tell me so I can repeat my tests.

**Comment by antonkatz** [comment]
Interestingly enough, in my own code (not this example) on my home Linux machine, page reloads don't seem to change the amount of memory that I see through `top`. But, doing a POST curl makes the memory grow by about 0.1M

**Comment by zhuxiujia** [comment]
> Interestingly enough, in my own code (not this example) on my home Linux machine, page reloads don't seem to change the amount of memory that I see through `top`. But, doing a POST curl makes the memory grow by about 0.1M

This means that the problem is with Windows and MacOS？

**Comment by medvednikov** [comment]
No, the problem is in V, I'm fixing it.

**Comment by smartiniOnGitHub** [comment]
Hi guys and thanks for the great work you're doing on the whole language.
I'm sorry to say that I'm repeating some tests with updated V on a minimal vweb example (you can find it [here](https://github.com/smartiniOnGitHub/vweb-example/blob/master/server.v)) and still I see memory grow each request (without and even *with* '-autofree'); just tried to understand something better with Valgrind and there are some memory leaks; sorry but can't say much more ... anyway tell me if I can help at least to provide more data.

**Comment by LouisSchmieder** [comment]
I think some tests are good, but it should be fixed soon

**Comment by kolesar-andras** [comment]
Memory still leaks in V 0.2.4 61b99e1.

**Comment by medvednikov** [comment]
Fixed in 0.3
